test/functional/ui/ext_win_spec.lua

```diff
@@ -57,4 +59,44 @@ describe('In ext-win mode', function()
     ]])
   end)
 
+  describe(':split', function()
+    it('sends correct events', function()
```

Comment:
@bfredl, should I be testing the events like this one? And if yes, how can I inspect the events?